docker-compose.yml

```diff
@@ -1,17 +1,19 @@
-web:
-  image: colinxfleming/dcaf_case_management
-  command: rails s -p 3000 -b 0.0.0.0
-  volumes:
-    - .:/usr/src/app
-  ports:
-    - "3000:3000"
-  links:
-    - db
-  environment:
-    mongohost: db
-  env_file:
-    - web-variables.env
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
+version: '3.1'
```

Comment:
🔥 🔥 🔥

Reply:
Yeah, a few versions back from "tip". Probably good enough for anyone needing to upgrade Docker.